commitlint.config.ts

```diff
@@ -300,10 +300,18 @@ module.exports = {
                     let message = `Please do not exceed ${maxLineLength} characters in title (found ${headerLength}).`;
                     if (!headerStr.startsWith('Merge ') && headerLength > maxLineLength) {
                         offence = true;
+
+                        let colonIndex = headerStr.indexOf(':')
+
+                        let title = headerStr
+                        if (colonIndex > 0){
+                            title = headerStr.substring(colonIndex)
```

Comment:
@realmarv a better name would be `titleWithoutArea`

Reply:
Done